src/rgw/rgw_rados.cc

```diff
@@ -339,10 +341,10 @@ class RGWDataNotifierManager : public RGWCoroutinesManager {
     for (auto iter = conn_map.begin(); iter != conn_map.end(); ++iter) {
       RGWRESTConn *conn = iter->second;
       RGWCoroutinesStack *stack = new RGWCoroutinesStack(store->ctx(), this);
-      stack->call(new RGWPostRESTResourceCR<bc::flat_map<int, bc::flat_set<string> >, int>(store->ctx(), conn, &http_manager, "/admin/log", pairs, shards, NULL));
-
+      stack->call(new RGWDataPostNotifyCR(store, http_manager, shards, pairs, conn));
```

Comment:
we're passing `rgw_http_param_pair* pairs` into `RGWDataPostNotifyCR` and using them for both `PostNotify2` and `PostNotify1`

but `pairs` includes `{ "notify", NULL }`, so both calls go to the old `RGWOp_DATALog_Notify` api. we'll need `PostNotify2` to use a separate array of `pairs` with `{ "notify2", NULL }` instead

let's just pass the `source_zone` into `RGWDataPostNotifyCR` instead of the `pairs`, and have `RGWDataPostNotifyCR` define `rgw_http_param_pair pairs[]` separately for each call

Reply:
sorry didn't notice this. Fixed. Thanks!

Reply:
will test backward compatibility again and update. Not ready to be merged yet.